ssl-impl/src/test/java/io/crate/protocols/postgres/ssl/SslConfigurationTest.java

```diff
@@ -87,7 +87,6 @@ public void testTrustStoreLoadingFail() throws Exception {
         settingsBuilder.put(SslConfigSettings.SSL_TRUSTSTORE_PASSWORD_SETTING_NAME, "wrongpassword");
 
         TrustStoreSettings.tryLoad(settingsBuilder.build());
-        fail();
```

Comment:
I have mixed feelings about modifying code which actually worked. I thought you were removing an important fail() at first.